app/views/shared/_article_promos.html.erb

```diff
@@ -8,7 +8,12 @@
       <div class="l-promo-articles__item">
         <div class="t-article-promos">
           <%= link_to item['url'], class: 'promo__link' do %>
-            <%= image_tag(item['image'], alt: '', class: 'promo__img') %>
+            <%= image_tag(item['image'],
+              alt: '',
+              sizes: '(max-width: 479px) 100vw, (max-width: 959px) 50vw, 25vw',
```

Comment:
@tomas-stefano it's used nowhere else but here. Where is the best place to put this or what is the best practice on handling this, please?